cert_test.go

```diff
@@ -40,11 +40,11 @@ func TestCertSerialize(t *testing.T) {
 func TestCertDeserialize(t *testing.T) {
 	// dig -t txt 2.dnscrypt-cert.opendns.com. -p 443 @208.67.220.220
 	b, err := unpackTxtString("DNSC\\000\\001\\000\\000\\200\\226E:H\\156\\203%\\134\\218\\127]\\168\\239\\027u\\011$\\191\\008\\239\\176F\\133\\017\\171\\161\\219\\154\\142i\\164\\010\\239\\017f\\168dS\\210f\\197\\194\\169\\171w\\2499\\1891\\155<\\130\\218@/\\155\\023v\\153#d\\024\\004\\136\\180\\228K5\\233d\\180\\144\\189\\218\\186\\232%\\162K\\004\\021\\160\\139\\225\\157}\\219\\135\\163<\\215~\\223\\142/qc78aWoo]\\221\\184`]\\221\\184`_\\190\\235\\224")
```

Comment:
Since you're improving tests (didn't know about `assert.NoError` btw, thanks!), this would probably be better off in a `testdata/` directory.  Or at least written with raw strings to prevent all these `\\`.